### DISCUSSION TITLE
"Clean all for this domain" does the job for the current container or all containers?

### DISCUSSION THREAD

**Discussion by pityalvarez** [Cookie-AutoDelete/Cookie-AutoDelete#1408] [Q&A / Support] [answered]
![image](https://user-images.githubusercontent.com/97750378/173447854-ef665080-cc86-4618-ab0f-d902ca0f5be0.png)
If so, how do I only clean the current container?

**Comment by kennethtran93** [comment, marked as answer]
Currently the manual cleanup actions clean the domain for all containers.  We have not implemented any of 'this domain' manual cleanups to only affect the specific container, primarily because the webextension API doesn't have it exposed/programmed for one specific container for all data types (according to the MDN docs, only cookies and indexedDB may be cleaned by the specific container, with localstorage if on Firefox nightly).

Source: 
- https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/browsingData/RemovalOptions
